src/applications/my-education-benefits/tests/form-submit-transform.unit.spec.js

```diff
@@ -0,0 +1,313 @@
+import { expect } from 'chai';
```

Comment:
Unit tests for each method in the transform utility functions to parse the form data and turn it into JSON.